app/assets/javascripts/classes/reports/EditReportViewModel.ts

```diff
@@ -232,14 +256,16 @@ class EditSourceViewModel extends BaseSourceViewModel {
 
     // Used by KO data-bind.
     private static readonly sourceTypeDisplayNames = {
-        [SourceType.WEB_PAGE]: "Web page",
+        [SourceType.WEB_PAGE]: "Web Page",
         [SourceType.GRAFANA]: "Grafana",
     };
     private static readonly sourceTypeHelp = {
-        [SourceType.WEB_PAGE]: "<li class='list-group-item'><b>Browser rendered</b> - A URL is loaded in the browser and the rendered " +
-                "contents of the page are taken as the generated report.</li>",
-        [SourceType.GRAFANA]: "<li class='list-group-item'><b>Grafana</b> - Like browser-rendered, but tweaked to pull data from a Grafana Report panel" +
-                " (which are loaded asynchronously after page-load, hence needing the specialization).</li>",
+        [SourceType.WEB_PAGE]: "<li class='list-group-item'><b>Browser rendered</b> - The specified URL is loaded in " +
+                "the browser and the page content is taken as the rendered report. This <i>only</i> supports static " +
+                "content.</li>",
+        [SourceType.GRAFANA]: "<li class='list-group-item'><b>Grafana</b> - Specialization of the Web Page source " +
+                "for capturing page content from Grafana based reports which may contain dynamic content. Additionally, " +
+                "these sources pass the scheduled date as the </li>",
```

Comment:
looks like this sentence got cut short

Reply:
Good catch. I was under the impression that we set from/to on the report that we generated based on the scheduled date, but it seems that we do not. I've cataloged this work in a document where I am tracking Reporting v1.1 work items.